### DISCUSSION TITLE
Change default keybinds to support IBus (and Caps Lock)

### DISCUSSION THREAD

**Discussion by AquilaFasciata** [zed-industries/zed#15061] [Feature Requests]
A little background, I use IBus on Wayland so I can type non-standard characters with my standard English keyboard. For whatever reason, IBus doesn't seem to pass through the shift key to other applications. Normally, this isn't an issue, as it seems that nvim checks for capital letters instead of shift+<key>. I think updating the assets/keymaps/vim.json file to change instances of shift to capitals when applicable. For instance `"shift-v": "vim::ToggleVisualLine"` would become `"V": "vim::ToggleVisualLine"`

I have done this with my most common commands and it seems to work perfectly. The only shortcoming would be a handful of shortcuts for items like `shift+tab` which don't have a capital form, though I think solving 98% of the problem is preferable to 0. 

This would solve my issue #14358 as well as another's #7512. I can make updates and submit a PR.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
